init/main.go

```diff
@@ -619,6 +622,15 @@ func boost() error {
 		return err
 	}
 
+	// Mount efivarfs if running in EFI mode
+	if _, err := os.Stat("/sys/firmware/efi"); errors.Is(err, os.ErrNotExist) {
```

Comment:
I believe it should be `!errors.Is(err, os.ErrNotExist)`. `if` body should be executed if the efi dir not-not exists (i.e. exists).